lib/routes/profile.js

```diff
@@ -60,7 +67,21 @@ module.exports = {
       } else {
         logger.info('batch.db');
       }
+
       return rep.header('last-modified', lastModified.toUTCString());
+    }
+
+    server.methods.profileCache.get(req, (err, result, cached, report) => {
+      // Check to see if the auth-server is reporting a newer `profileChangedAt`
+      // than the cached value. If so, lets invalidate the cache and set new value.
```

Comment:
This comment could be a little misleading - it's not the auth-server that's reporting the newer timestamp in this case, it's the oauth-server from validating the token.